scripts/02-setup-kpm.sh

```diff
@@ -6,13 +6,13 @@ function log() {
 }
 SELF_DIR=$(git -C $(dirname $0) rev-parse --show-toplevel)
 
-if ! [ -f "$SELF_DIR/patch_linux" ]; then
-    log "No patch_linux! downloading..."
+if ! [ -f "scripts/patch_kpm" ]; then
```

Comment:
Is it better to place patch_kpm in the root directory of the kernel source? Should we also add patch_kpm to kernel tree .gitignore file?

Reply:
The reason why we shouldn't commit any binary files is that they are not diffable, which means if you change it, git will store a completely new copy of it. However, I agree with you this time because patch_kpm binary is small and we don't often have updates.